gradle.properties

```diff
@@ -15,9 +15,7 @@
 #
 org.gradle.parallel=true
 
-kotlinVersion=1.2.41
-junitVersion=1.2.0
-jupiterVersion=5.0.2
-junitLegacyVersion=4.12.0
-spekVersion=1.1.5
-keikoVersion=2.9.8
+enablePublishing=false
+
+keikoVersion=2.10.1
```

Comment:
Why are these in gradle.properties and not in build.gradle? (keiko and fiat)

Reply:
Yeah, I meant to move those. Doing it nao.